packages/ui/src/components/Settings/SettingsForm.test.tsx

```diff
@@ -14,23 +15,27 @@ describe('SettingsForm', () => {
 
   it('should render', () => {
     const wrapper = render(
-      <ReloadContext.Provider value={{ reloadPage, lastRender: 0 }}>
-        <SettingsProvider adapter={settingsAdapter}>
-          <SettingsForm />
-        </SettingsProvider>
-      </ReloadContext.Provider>,
+      <MemoryRouter>
```

Comment:
It seems this change disturbed sonarcloud and let it complain about an existing duplication. Maybe create a private function to create this React element and let these test cases use it?

Reply:
Thanks for the comment @igarashitm. I'll fix this repetition of code!